epcpy/utils/parsers.py

```diff
@@ -66,6 +67,23 @@
 }
 
 
+def ignore_errors(func: Callable[..., Any], *args, **kwargs) -> Any:
```

Comment:
I just can't see why getting None is better than getting an appropriate exception for the user. In fact, getting None worse I would say because it's much less informative.   

We shouldn't provide such things and if the user of this library finds this convenient for the task at hand, then let the user implement such a function.